examples/chaiscript/CMakeLists.txt

```diff
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.2)
+
+# Emulate HunterGate:
+# * https://github.com/hunter-packages/gate
+include("../common.cmake")
+
+project(download-ChaiScript)
+
+# DOCUMENTATION_START {
+hunter_add_package(chaiscript)
+find_package(chaiscript CONFIG REQUIRED)
+# DOCUMENTATION_END }
```

Comment:
Please refactor this part so `target_link_libraries` will be part of documentation too. Users need to know what imported targets this package provides.